### PR TITLE
F upgrade storybook migrate dp button

### PR DESCRIPTION
**Pull request by ahmad-demos**
https://github.com/demos-europe/demosplan-ui/pull/379

- Migrate DpButton stories to v7.

### CODE REVIEW

File: src/components/DpButton/DpButton.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        color: 'secondary',
+        hideText: false,
+        icon: 'delete',
+        iconSize: 'medium',
+        rounded: true,
+        text: 'Delete',
+        type: 'submit',
+        variant: 'outline'
+    }
+}
```
Comment (salisdemos) [inline src/components/DpButton/DpButton.stories.tsx:25]: shouldn't be the Default quite "default". Here You set several props to not so default values ;-)

**Review by salisdemos — CHANGES_REQUESTED** [review]

File: src/components/DpButton/DpButton.stories.tsx (hunk 1 above)
Comment (ahmad-demos) [reply src/components/DpButton/DpButton.stories.tsx]: https://github.com/demos-europe/demosplan-ui/pull/397/commits/3b0f3d620777ef82c0b2777830607ffa6910e3d5

Comment (salisdemos) [reply src/components/DpButton/DpButton.stories.tsx]: but why do You define them at all? can't You just omit the props (they are all not required)

Comment (ahmad-demos) [reply src/components/DpButton/DpButton.stories.tsx]: sorry, yes we can
https://github.com/demos-europe/demosplan-ui/pull/397/commits/a0fcebfe307df57f1b0ad9c37a874cbb69d2d5d1

thanks

**Review by salisdemos — APPROVED** [review]

**Pull request merged**